CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+## 2.0.0-SNAPSHOT
+
+- Adds support for new reference types in FaunaDB API 2.1
+  - `DatabaseV`, `ClassV`, `IndexV`, `FunctionV` and `KeyV`
+- Removes `Ref(string id)` function
+  - All refs from now on should be created using constructors like `new RefV(...)` or `new DatabaseV(...)`
```

Comment:
Yeah, it's still available. Will just add more test on top of the function. I'm just testing serialization